klarna-on-demand/src/main/java/com/klarna/ondemand/RegistrationSettingsBuilder.java

```diff
@@ -0,0 +1,40 @@
+package com.klarna.ondemand;
+
+/**
+ * Builds a RegistrationSettings object
+ */
+public class RegistrationSettingsBuilder {
```

Comment:
I really don't feel that the `RegistrationSettings` object is complicated enough to warrant a builder.
Convince me otherwise.